examples/counter/android/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Curious on what your thoughts are on using ConstraintLayout. I understand it's not really the point of this app, but it'd be nice to use best practices in general

Reply:
Love constraintlayout and use it for almost everything, just was a bit quicker for me to do it this way.  Can we get this merged and do another PR for constraintlayout?